xml/System/Environment.xml

```diff
@@ -666,21 +666,36 @@ goto :EOF
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Environment.GetEnvironmentVariable%28System.String%29> method retrieves an environment variable from the environment block of the current process only. It is equivalent to calling the <xref:System.Environment.GetEnvironmentVariable%28System.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>. The environment block of the current process includes the following environment variables:  
+
+The <xref:System.Environment.GetEnvironmentVariable%28System.String%29> method retrieves an environment variable from the environment block of the current process only. It is equivalent to calling the <xref:System.Environment.GetEnvironmentVariable%28System.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>. 
+
+To retrieve all environment variables along with their values, call the <xref:System.Environment.GetEnvironmentVariables%2A> method.  
   
--   All per-machine environment variables that are defined at the time the process is created, along with their values.  
-  
--   All per-user environment variables that are defined at the time the process is created, along with their values.  
-  
--   Any variables added to the process block while the process is running by calling either the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%29> method or the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>.  
-  
- If environment variables are created after the process has started, you can use this method to retrieve only those variables that were created by calling the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%29> method or the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of .<xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>.  
-  
- To retrieve all environment variables along with their values, call the <xref:System.Environment.GetEnvironmentVariables%2A> method.  
-  
- Environment variable names are not case-sensitive.  
-  
-   
+Environment variable names are case-sensitive on Linux and macOS but are not case-sensitive on Windows.  
+ 
+### On Windows systems
+
+ On Windows systems, the environment block of the current process includes the following environment variables:  
+  
+- All per-machine environment variables that are defined at the time the process is created, along with their values.  
```

Comment:
These two points should be deleted even for Windows. They are not accurate statements.

> All per-machine environment variables that are defined at the time the process is created, along with their values.  
> All per-user environment variables that are defined at the time the process is created, along with their values.

Instead, I would rephrase the third point to be something like this:

All environment variables that are provided to it by the parent process that created it. For example, .NET application launched from a Console window inherits all of the Console window's environment variables.

If there is no parent process, per-machine and per-user environment variables are used instead. For example, a new Console window has all per-machine and per-user environment variables defined at the time it was launched.

Reply:
Thanks for your review, @jkotas. I've made the modification as you suggested.